packages/sdk-codegen/src/python.gen.ts

```diff
@@ -441,11 +487,10 @@ ${this.hooks.join('\n')}
     const attrs: string[] = []
     const isEnum = type instanceof EnumType
     const baseClass = isEnum ? 'enum.Enum' : 'model.Model'
-    let attrsArgs = 'auto_attribs=True, kw_only=True'
```

Comment:
now it's always `auto_attribs=True, init=False` so no need to have a separate variable